src/libs/handlerResolver.ts

```diff
@@ -1,3 +1,3 @@
 export const handlerPath = (context: string) => {
```

Comment:
what's this for?

Reply:
This function was the provided in the Serverless TypeScript template and it's used here:
https://github.com/Open-Attestation/api.verify.gov.sg/pull/1/files/b8cc250abb627f7bf09b26899eda98cd185793ae#diff-1e74c0a60982d61a8b05235e3d1b01b88dba242446c9f49f1299393ceb11202cR5

Reply:
ok, can add a comment to explain what it does?